projects/telemetry/inc/nmea.h

```diff
@@ -0,0 +1,25 @@
+// # The NMEA message format
+// A given NMEA message starts with $, then contains various comma-seperated values,
+// followed by a *, and then the message checksum (which is a 2-digit hex number).
+// This header will aid in parsing the NMEA messages sent by the GPS.
+
+#pragma once
```

Comment:
By convention `#pragma once` should be the first line. The usual format is:

```
#pragma once
// module summary
// requires ...
// module comments/description
#include ...
```

Reply:
Thanks for the heads up, moved it